### DISCUSSION TITLE
How many audio streaming connections are supported under one account at the same time?

### DISCUSSION THREAD

**Discussion by AnnaPi03** [deepgram/community#81] [General help] [answered]
How many audio streaming connections are supported under one account at the same time?

**Comment by BekahHW** [comment, marked as answer]
@AnnaPi03 are you asking how many concurrent streams can happen under one project ID?

**Reply by AnnaPi03** [reply]
Yes, exactly

**Reply by BekahHW** [reply]
The number of concurrent connections allowed by default will be changing in the near future. If you’re looking to use Deepgram’s real-time ASR at scale, you can reach out to our sales team for more info. But if you want to provide more context, I'm happy to help.
